ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_influx.schema.yml

```
@@ -0,0 +1,74 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_telemetry_influx:
+    type: dict
+    keys:
+      vrf:
+        type: str
+      destinations:
+        type: list
+        primary_key: name
+        description: Configure telemetry output destinations.
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: InfluxDB connection name.
+            database:
+              type: str
+              description: Set name of the database.
+            data_retention_policy:
+              type: str
+            url:
+              type: str
+            username:
+              type: str
+            password:
+              type: str
+            password_type:
+              type: str
+              convert_types:
+                - int
+              valid_values: ["0", "7", "8a"]
+              default: "7"
```

```suggestion
              default: "7"
Default should be 0
since 7 only takes encrypted value, EOS cli converts 0 to 7.
```suggestion
              default: "0"
```

```
username aaa password 7 password
% Invalid input
```

---

fixed

---

No, I asked for this to be 7 since that is what we do elsewhere in eos_cli_config_gen.

---

Okay, then we can may be write in the description that we need to provide encrypted value for `password_type` in case of default value.

---

> No, I asked for this to be 7 since that is what we do elsewhere in eos_cli_config_gen.

Reverted to 7